src/main/webapp/Measure.js

```diff
@@ -0,0 +1,72 @@
+/*
+This is the measure class, which represents a 
+music unit. You have to create it with a Note
+and you can play this note.
+
+
+<div class="grid-item">
+    <div class="dropdown">
+        <button onclick="showMenu()" class="dropButton">Notes</button>
+        <div id="myDropdown" class="dropdown-content">
+            <a href="#one">One</a>
+            <a href="#two">Two</a>
+            <a href="#three">Three</a>
+        </div>
+    </div>
+</div>
+
+*/
+
+class Measure {
+    constructor(parentDiv) {
+        //Create big dropdown div
+        var dropDownDiv = document.createElement("div");
+        dropDownDiv.classList.add("dropdown");
```

Comment:
Given that you create the element right before assigning the class, the element would not start off with any classes. I would assign the class using .className instead of .classList.add() just for simplicity. 

However, if there is a particular advantage to using .classList.add() rather than .className, then please disregard this comment :)